WebRoot/js/zimbraMail/briefcase/view/ZmDetailListView.js

```diff
@@ -362,6 +362,39 @@ function(item) {
 	}
 };
 
+ZmDetailListView.prototype._getLabelFieldList =
+function () {
+	var headers = [];
+	headers.push(ZmItem.F_SELECTION);
+	headers.push(ZmItem.F_NAME);
+	headers.push(ZmItem.F_TYPE);
+	headers.push(ZmItem.F_SIZE);
+	headers.push(ZmItem.F_DATE);
+	headers.push(ZmItem.F_SHARES);
+	return headers;
+};
+
+ZmDetailListView.prototype._getLabelForField =
+function(item, field) {
+	switch (field) {
+		case ZmItem.F_NAME:
+			return item.name;
+		case ZmItem.F_TYPE:
+			var itemInfo = ZmMimeTable.getInfo(item.contentType);
+			return itemInfo.desc;
```

Comment:
would there be possibility of itemInfo having null or undefined value?

Reply:
Addressed.